shared/constants/network.ts

```diff
@@ -1116,3 +1116,10 @@ export const TEST_NETWORKS = [
   LINEA_GOERLI_DISPLAY_NAME,
   LINEA_SEPOLIA_DISPLAY_NAME,
 ];
+
+export const TEST_NETWORK_IDS = [
```

Comment:
I didn't want to compare `nickname` as previous selectors do it hence adding testnet ids here.

Reply:
To clarify, are you saying there is no existing constant of test chain IDs, only the above names?

Reply:
Exactly, I couldn't find any map or an array in the extension code.